builder/api/src/main/java/module-info.java

```diff
@@ -17,8 +17,9 @@
 /**
  * The Builder API module.
  */
-module io.helidon.builder {
-    requires static io.helidon.config.metadata;
+module io.helidon.builder.api {
+    requires transitive io.helidon.common;
```

Comment:
No need for static metadata inclusion?

Reply:
do you see a reason to include it?

The choice whether to use anything from metadata is on the user. The annotations are not required to code generate a builder.